src/bindings/python/src/compatibility/pyngraph/CMakeLists.txt

```diff
@@ -85,6 +82,4 @@ if(OpenVINO_SOURCE_DIR OR InferenceEngineDeveloperPackage_FOUND)
     install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../../../tests
             DESTINATION tests/${PROJECT_NAME}
             COMPONENT tests EXCLUDE_FROM_ALL)
-
-    ie_cpack(pyngraph_${PYTHON_VERSION})
```

Comment:
if it removes cpack generation for this compoent please revert this line. This cpack is used for conda packages.

Reply:
`ie_cpack` should be called once per project. We already have `ie_cpack` call in python bindings root folder https://github.com/openvinotoolkit/openvino/blob/96e345d81ed1564ba39f1ed2ece31a5f0ab6bcfc/src/bindings/python/CMakeLists.txt